package.json

```diff
@@ -930,6 +939,11 @@
         "view": "openshiftComponentsView",
         "contents": "You have no OpenShift Components in your workspace yet.\n[New Component](command:openshift.component.createFromLocal)",
         "enablement": "openshift.component.explorer.init"
+      },
+      {
+        "view": "openshiftComponentTypesView",
+        "contents": "Default Devfile Regisitries are Here.\n[Open Default Devfile Registry](command:openshift.component.open.devfileRegistry)",
```

Comment:
This does not fit Component Types View purpose. Welcome contents appears only when view has no items. It means when there is no registries listed with 'odo registry list' it supposed to show welcome content. The button should trigger 'Add registry command' not 'Open Default registry'.